questions.js

```diff
@@ -72,12 +72,11 @@ module.exports = [
       {value: 'cypress', title: 'Yes (Cypress)', hint: 'Cypress offers fast, easy and reliable testing for anything that runs in a browser.'}
     ]
   },
-  // This question is not raised to end user, since there is only one choice, it will be selected automatically.
-  // We will add more sample code in future.
   {
     message: 'What kind of sample code do you want in this project?',
     choices: [
-      {value: 'app-min', title: 'Bare minimum'}
+      {value: 'app-min', title: 'Bare minimum'},
+      {value: 'skeleton-navigation', title: 'Skeleton Navigation', hint: 'Application with Navigation setup using conventional routing'}
```

Comment:
Pls rename it to `app-with-router` (following `app-min`). The old name from aurelia-cli is bit too verbose.

Reply:
done